src/controller.coffee

```diff
@@ -1853,13 +1878,13 @@ define ['droplet-helper',
       # value.
       unless @textFocus.handwritten
         newParse = null
-        string = @textFocus.stringify(@mode.empty).trim()
+        string = @mode.normalString @textFocus.stringify(@mode.empty)
```

Comment:
might we want all the logic from line 1881 to 1890 to be part of the base Parser normalString, i.e., trim and look at quotes?  Then specific modes can override to have their whole own idea of what quotes are etc.